packages/react-dev-utils/webpackHotDevClient.js

```diff
@@ -239,7 +239,7 @@ function canAcceptErrors() {
   // However, when hot-reload status is abort or fail,
   // it indicates the current update cannot be applied safely,
   // and thus we should bail out to a forced reload for consistency.
-  return hasReactRefresh && ["abort", "fail"].indexOf(status) === -1
+  return hasReactRefresh && ['abort', 'fail'].indexOf(status) === -1;
```

Comment:
```suggestion
  return hasReactRefresh && !['abort', 'fail'].includes(status);
```

Reply:
This part of the code was touched by prettier I'm trying to keep this PR to a minimum - let's create a separate issue for setting up proper eslint code style conventions, with auto fix etc. - good input but resolving for now - pushing for getting the alpha out.

I would likely do `['abort', 'fail'].includes(status) === false` as the `!` is often missed in code reviews (eg. readability) - but I'm sure it will be a lengthly oppinionated discussion :)

Reply:
@elektronik2k5 I've defered the discussion to #11274 - I do agree that it's something we should talk about